Update Review form

Chandu6702 opened the pull request:
## Related Issue
closes: #1021 

## Description
Text in the review is not appearing to the users so I fixed it by changing the font styling like font size, color, weight, letter spacing, etc and also updated the CSS of the review form for optimal user experience.

## Type of PR

- [x] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
before:
![Screenshot 2024-06-22 133511](https://github.com/HimanshuNarware/Devlabs/assets/125206920/cf605940-8fc8-4921-9996-4db3d0d6f6ae)
![Screenshot 2024-06-22 134839](https://github.com/HimanshuNarware/Devlabs/assets/125206920/13a3e9b9-9c81-49dc-b93f-f36b805c5be6)


After:

https://github.com/HimanshuNarware/Devlabs/assets/125206920/d8726b8f-9dd6-4f27-b8b9-bee33d5a6147



## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.
<!-- [X] - put a cross/X inside [] to check the box -->

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

Chandu6702 commented:
@panwar8279 please review and merge this PR

panwar8279 commented:
Hi @Chandu6702, color combination is not looks good. Current color combination is much better.

Chandu6702 commented:
@panwar8279 so should I change it to default colour combination

panwar8279 commented:
@Chandu6702 , yes

Chandu6702 commented:
@panwar8279, I have changed the color combination to default and only changed the font styles. I have added font weight, size, letter spacing, border-radius, etc. for optimal view for the user.

https://github.com/HimanshuNarware/Devlabs/assets/125206920/e73301bd-afb0-4c2a-811b-f1378a2d4413

Please review and merge the PR.

panwar8279 commented:
@Chandu6702 , Please solve file conflict issue.
